src/Microsoft.AspNetCore.WebUtilities/BufferedReadStream.cs

```diff
@@ -202,6 +235,10 @@ public override int Read(byte[] buffer, int offset, int count)
             return await _inner.ReadAsync(buffer, offset, count, cancellationToken);
         }
 
+        /// <summary>
+        /// Ensures that the buffer is not empty
+        /// </summary>
+        /// <returns>buffer has data</returns>
```

Comment:
For these types of docs, you can do something similar to this: https://github.com/aspnet/Mvc/blob/760c8f38678118734399c58c2dac981ea6e47046/src/Microsoft.AspNetCore.Mvc.Abstractions/Formatters/IOutputFormatter.cs#L18

```
        /// <returns>Returns <c>true</c> if the formatter can write the response; <c>false</c> otherwise.</returns>
```